### PR TITLE
Code Review - Change

### PR DESCRIPTION
**Pull request by Thumimku**
## Purpose
This PR contains results suggested in code Review -1

- Elaborated Javadoc for the public methods in Common
- Remove tenant details from service interface.

### CODE REVIEW

File: components/org.wso2.carbon.identity.remotefetch.common/src/main/java/org/wso2/carbon/identity/remotefetch/common/ConfigurationFileStream.java [hunk 1]
```diff
@@ -22,14 +22,15 @@
...
+ * Holds raw config file content for reading xml files to deploy configurations.
+ *
  */
 public class ConfigurationFileStream {
 
     private InputStream contentStream;
     private File path;
 
-    /**
+    /**Default constructor.
```
Comment (JKAUSHALYA) [inline components/org.wso2.carbon.identity.remotefetch.common/src/main/java/org/wso2/carbon/identity/remotefetch/common/ConfigurationFileStream.java:33]: Formatting issue. /** line should be kept without any other text. Please fix all the instances.

Comment (Thumimku) [reply components/org.wso2.carbon.identity.remotefetch.common/src/main/java/org/wso2/carbon/identity/remotefetch/common/ConfigurationFileStream.java]: Fixed

File: components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/dao/RemoteFetchConfigurationDAO.java [hunk 2]
```diff
@@ -72,14 +66,15 @@
      * @return
      * @throws RemoteFetchCoreException
      */
-    List<RemoteFetchConfiguration> getRemoteFetchConfigurationsByTenant(int tenantId) throws RemoteFetchCoreException;
+    List<RemoteFetchConfiguration> getRemoteFetchConfigurationsByTenant(int tenantId)
+            throws RemoteFetchCoreException;
 
     /**
-     * @param tenantId
+     * @param tenantDomain
```
Comment (JKAUSHALYA) [inline components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/dao/RemoteFetchConfigurationDAO.java:73]: Add a proper JavaDoc comment.

File: components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/dao/impl/RemoteFetchConfigurationDAOImpl.java [hunk 3]
```diff
@@ -193,14 +180,15 @@ public void deleteRemoteFetchConfiguration(int configurationId) throws RemoteFet
     }
 
     /**
-     * @param tenantId
+     * @param tenantDomain
```
Comment (JKAUSHALYA) [inline components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/dao/impl/RemoteFetchConfigurationDAOImpl.java:183]: Add a proper JavaDoc comment.

File: components/org.wso2.carbon.identity.remotefetch.common/src/main/java/org/wso2/carbon/identity/remotefetch/common/BasicRemoteFetchConfiguration.java [hunk 4]
```diff
@@ -51,46 +52,85 @@ public BasicRemoteFetchConfiguration(int id, boolean isEnabled,
...
     public int getFailedDeployments() {
 
         return failedDeployments;
     }
 
+    /**
+     * get last deployed time for listview.
+     * @return lastDeployed
+     */
     public Date getLastDeployed() {
```
Comment (brionmario) [inline components/org.wso2.carbon.identity.remotefetch.common/src/main/java/org/wso2/carbon/identity/remotefetch/common/BasicRemoteFetchConfiguration.java:134]: Keep a space after the method name. Check other places as well.

Comment (Thumimku) [reply components/org.wso2.carbon.identity.remotefetch.common/src/main/java/org/wso2/carbon/identity/remotefetch/common/BasicRemoteFetchConfiguration.java:134]: Thank you, This comment and your previous comments were fixed in
https://github.com/wso2-extensions/identity-fetch-remote/pull/40
https://github.com/wso2-extensions/identity-fetch-remote/pull/41
https://github.com/wso2-extensions/identity-fetch-remote/pull/42
https://github.com/wso2-extensions/identity-fetch-remote/pull/43
https://github.com/wso2-extensions/identity-fetch-remote/pull/44
https://github.com/wso2-extensions/identity-fetch-remote/pull/45.
I resolve this conversation.

File: components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/dao/impl/RemoteFetchConfigurationDAOImpl.java (hunk 3 above)
Comment (Thumimku) [reply components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/dao/impl/RemoteFetchConfigurationDAOImpl.java:183]: Thank you, This comment and your previous comments were fixed in
https://github.com/wso2-extensions/identity-fetch-remote/pull/40
https://github.com/wso2-extensions/identity-fetch-remote/pull/41
https://github.com/wso2-extensions/identity-fetch-remote/pull/42
https://github.com/wso2-extensions/identity-fetch-remote/pull/43
https://github.com/wso2-extensions/identity-fetch-remote/pull/44
https://github.com/wso2-extensions/identity-fetch-remote/pull/45.
Hence I resolve this conversation.

**Pull request merged**